README.md

```diff
@@ -140,3 +141,17 @@ Start deployment
 ```bash
 ./dobi.sh cpu01-edgefarm-minio-deploy
 ```
+
+### Install Raspberry Pi sdimg on SD Card
+
+Find build images:
+
+```
+ls */install/images/raspberrypi4-64/core-image-base-raspberrypi4-64-*.sdimg
```

Comment:
The filename is something like EdgeFarm-Devtools-Image-raspberrypi4-64-20220224142829.sdimg